easybuild/easyconfigs/p/pod5-file-format/pod5-file-format-0.1.8_dep_fix.patch

```diff
@@ -0,0 +1,28 @@
+diff -u pod5-file-format-0.1.8/c++/CMakeLists.txt.orig pod5-file-format-0.1.8/c++/CMakeLists.txt
```

Comment:
@PetrKralCZ Please include a short description on top of the patch file to describe what it does, and mention the authors.
Note that this implies also updating the SHA256 checksum of the patch file